src/cli.js

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env node
+
+'use strict'
+
+const YargsPromise = require('yargs-promise')
+const yargs = require('yargs')
+const process = require('process')
+const log = require('debug')('repo-migrations:cli')
+
+const commands = require('./commands')
+
+function print (msg = '', newline = true) {
+  if (newline === undefined) {
```

Comment:
My suggestion to change the line above from
`print (msg, newline)` => `print (msg = '', newline = true)`
was so that you could remove lines 13 - 19

Reply:
😆 yeah I did not realize that, I just blindly accepted your suggestion. Will fix.